### PR TITLE
Updating the variable tags

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/audiobuffer/duration/index.md [hunk 1]
```diff
@@ -24,20 +24,20 @@ A double.
...
 // Stereo
-var channels = 2;
+const channels = 2;
 
 // Create an empty two second stereo buffer at the
 // sample rate of the AudioContext
-var frameCount = audioCtx.sampleRate * 2.0;
-var myArrayBuffer = audioCtx.createBuffer(2, frameCount, audioCtx.sampleRate);
+let frameCount = audioCtx.sampleRate * 2.0;
+let myArrayBuffer = audioCtx.createBuffer(2, frameCount, audioCtx.sampleRate);
```
Comment (wbamberg) [inline files/en-us/web/api/audiobuffer/duration/index.md:32]:
```suggestion
const frameCount = audioCtx.sampleRate * 2.0;
const myArrayBuffer = audioCtx.createBuffer(2, frameCount, audioCtx.sampleRate);
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for your PR! I think a couple of those `let` declarations could be `const`.

**Comment by wbamberg** [comment]
Also, please follow the instructions in https://github.com/mdn/content/issues/16662#issue-1248441427, or it gets really hard to keep track of the changes.

**Comment by gullapalli9482** [comment]
I knew this would happen.
It's just for this particular file, I'll group them for the next pr.
Thank you for the suggestion with the variables, I knew those two were meant to be const somehow ended up declaring them let.

**Review by wbamberg — APPROVED** [review]
👍 looks great, thank you @gullapalli9482 !

**Pull request merged**